hips/hip-0002.md

```diff
@@ -0,0 +1,63 @@
+
+As it is not recommended to use Helm with a version of Kubernetes that is newer than the version Helm was compiled against ([see here][helm-skew]), there is value in aligning the date for Helm minor releases with [Kubernetes releases][kubernetes-dates], which state that "X.Y.0 occur 3 to 4 months after X.(Y-1).0".
+
+Helm major releases happen rarely and therefore do not lend themselves to pre-defined, cyclical release dates.  However, upon the availability of the first beta version of a major release, a specific final release date should be chosen and announced.
+
+## Specification
+
+Patch releases should be done once a month on the second Wednesday of each month (assuming there are changes since the last release). A patch release to fix a high priority regression or security issue does not have to follow this schedule, but it is highly desirable that it is released on a Wednesday if possible.
+
+For minor releases, it is not necessary to aim for dates that are the same every year.  Instead, the following is proposed:
+* Once a Helm minor release is made, the release date of the next minor release must be announced.
+* To align with Kubernetes releases, a 4-month minor release cadence should be adopted (so at least 3 minor releases of Helm per year)
+
+Extra minor releases can be added to the schedule when needed.  However:
+* An extra minor release should not change a planned minor release date, unless the planned release is less than 7 days away.  This is to avoid postponing the release of features that are still on-going and planned according to the originally stated release date.
+* Extra minor releases should only be done for important reasons (as per judgment of the maintainers) to avoid increasing the burden on organizations that choose to upgrade at every release.
+
+## Security implications
+
+Security releases do not follow any planned dates and should be done as soon as required.
+
+## How to teach this
+
+* When a Helm minor release is made, the next minor release number and date should be chosen and announced at the same time.
+* The Helm website should have a page mentioning each planned release and its date.  This page should be updated at each release.
+
+## Reference implementation
+
+A reference implementation for this proposal to be accepted would entail:
+* Proposing a date of the next minor release
+* Posting a PR to helm-www documenting the new process and the chosen date
+* Posting a PR to helm-www updating the release check-list to include the step of selecting the next minor release date and publishing it
+
+## Open issues
+
+* This HIP proposes that for minor releases, only the date of the next release will be announced.  We could instead have fixed dates such as: "The first Wednesday of February, June and October".  What approach is preferred?
+* How many previous versions of Helm receive patch releases? If more than one, should those patch releases be on the same date?
+
```

Comment:
What should I do with the "open issues" section once there are enough LGTM?
I'm assuming these are no longer open issues since they didn't affect approval.
Should I remove the section?

Reply:
I've removed the "Open issues" section since we settled on a way forward.